src/ontology/oba.Makefile

```diff
@@ -139,14 +160,28 @@ $(REPORTDIR)/%.tsv: ../sparql/synonyms-exact.sparql $(TMPDIR)/%.owl
 prepare_oba_alignment: $(REPORTDIR)/uberon.tsv $(REPORTDIR)/efo.tsv $(REPORTDIR)/pato.tsv $(REPORTDIR)/oba.tsv $(REPORTDIR)/vt.tsv $(REPORTDIR)/cl.tsv $(REPORTDIR)/go.tsv $(REPORTDIR)/chebi.tsv
 	echo "OK cool all tables prepared."
 
-CHECK_SPARQL=oba.owl
+ONTOLOGIES=vt oba
+ONTOLOGIES_DIR=data/ontologies
+OWL_FILES=$(foreach o,$(ONTOLOGIES), $(ONTOLOGIES_DIR)/$(o).owl)
+
+DB_FILES=$(foreach o,$(ONTOLOGIES), $(ONTOLOGIES_DIR)/$(o).db)
+OWL_SOURCES=$(patsubst $(ONTOLOGIES_DIR)/%.owl, http://purl.obolibrary.org/obo/%.owl,$@)
+
+$(OWL_FILES):
+	@[ -d $(ONTOLOGIES_DIR) ] || mkdir -p $(ONTOLOGIES_DIR)
+	@wget $(OWL_SOURCES) -P $(ONTOLOGIES_DIR)
+
+
+$(DB_FILES): $(OWL_FILES)
```

Comment:
Don't define these goals in terms of multiple files. Better have single file goals:

```
$(TMPDIR)/%:
	wget http://purl.obolibrary.org/obo/$*.owl -O $@

$(TMPDIR)/semseql_%.db: $(TMPDIR)/%
	@semsql make $@
```

Reply:
I've tried to do in that way, but the percent sign isn't be replaced.

```
$(ONTOLOGIES_DIR)/%:
	@[ -d $(ONTOLOGIES_DIR) ] || mkdir -p $(ONTOLOGIES_DIR)
	@ echo "wget http://purl.obolibrary.org/obo/$*.owl -O $@"
```
Output:
`wget http://purl.obolibrary.org/obo/%.owl -O data/ontologies/%`

Reply:
Could this be a conseqyebce of your use of quotes? Maybe this makes `make` interpret the $* somehow differently. Also, just don't use the `@` suppressors during development in the beginning of each line.